### PR TITLE
Resolves #2075

### LINKED ISSUE #2075: New-ADFineGrainedPasswordPolicy - UnClear How To Lockout duration to Administrator unlock
It is not clear how to set a lockout policy that requires the adminsitrator to unlock the account.

This appears to work however when you look at the created policy in the Active Directory Administrative centre the requred field "Reset failed logon attempts count after (mins)" is empty.

New-ADFineGrainedPasswordPolicy -name "HighlyPrivilegedGroups1111" -Precedence 10 -MinPasswordLength 15 -PasswordHistoryCount 24 -ComplexityEnabled $true -ReversibleEncryptionEnabled $false -ProtectedFromAccidentalDeletion $true -MinPasswordAge 1 -MaxPasswordAge 42 -LockoutDuration "00:00:00" -LockoutThreshold 3 -LockoutObservationWindow "00:00:00"

### PR DESCRIPTION
**Pull request by velkovb**
Add example for manual unlock
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2075

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR.

cc @JohanFreelancer9 for copy editing.

Thank you.

File: docset/windows/addsadministration/New-ADFineGrainedPasswordPolicy.md [hunk 1]
```diff
@@ -239,7 +246,7 @@ Accept wildcard characters: False
 
 ### -LockoutDuration
 Specifies the length of time that an account is locked after the number of failed login attempts exceeds the lockout threshold.
-You cannot login to an account that is locked until the lockout duration time period has expired.
+You cannot login to an account that is locked until the lockout duration time period has expired. If you set the value to 0 the account needs to be unlocked manually by administrator.
```
Comment (dariomws) [inline docset/windows/addsadministration/New-ADFineGrainedPasswordPolicy.md:249]:
```suggestion
You cannot log in to an account that is locked until the lockout duration time period has expired. If you set the value to 0 the account needs to be unlocked manually by the administrator.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by get-itips — COMMENTED** [review]
i am ok once suggestions are applied

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md [hunk 2]
```diff
@@ -697,16 +529,11 @@ Accept wildcard characters: False
 
 ### -ReversibleEncryptionEnabled
 Specifies whether the directory must  store passwords using reversible encryption.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md:531]:
```suggestion
Specifies whether the directory must store passwords using reversible encryption.
```

File: docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md [hunk 3]
```diff
@@ -722,43 +549,26 @@ Accept wildcard characters: False
 
 ### -Server
 Specifies the Active Directory Domain Services instance to connect to, by providing one of the following values for a corresponding domain name or directory server.
-The service may be any of the following:  Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory Snapshot instance.
+The service may be any of the following:  Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory snapshot instance.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md:552]:
```suggestion
The service may be any of the following: Active Directory Lightweight Domain Services, Active Directory Domain Services, or Active Directory snapshot instance.
```

File: docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md [hunk 4]
```diff
@@ -27,78 +27,71 @@ New-ADFineGrainedPasswordPolicy [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-
...
+This example creates two new fine-grained password policy objects using a template object.
 
------------
+### Example 3: Create a fine-grained password policy with manual account unlock
+```powershell
+PS C:\> New-ADFineGrainedPasswordPolicy -Name "ManualUnlockPSO" -Precedence 500 -ComplexityEnabled $true -Description "Manual Unlock Password Policy" -DisplayName "Manual Unlock PSO" -LockoutDuration "00:00:00" -LockoutObservationWindow "00:00:00" -LockoutThreshold 3
+```
 
-Create two new Fine Grained Password Policy object using a template object.
+This command creates a fine-grained password policy object named ManualUnlockPSO that would require manual unlock of accounts by administrator.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/New-ADFineGrainedPasswordPolicy.md:80]:
```suggestion
This command creates a fine-grained password policy object named ManualUnlockPSO that would require manual unlock of accounts by the administrator.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - three more minor edits.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**